src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/Example2.java

```diff
@@ -0,0 +1,41 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MethodLength">
+      <property name="tokens" value="METHOD_DEF"/>
+      <property name="max" value="4"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.sizes.methodlength;
+
+// xdoc section -- start
+public class Example2 {
+
+  public Example2()  {         // constructor (line 1)
+    int var1 = 2;           // line 2
```

Comment:
Please remove all `// line 3` type comments in all examples, these really don't provide any value when we set max to a low number. Lines are easily countable in this case.

Reply:
done